Meta/run.sh

```diff
@@ -304,12 +304,14 @@ if [ -z "$SERENITY_MACHINE" ]; then
         SERENITY_MACHINE="
         -m $SERENITY_RAM_SIZE
         -smp $SERENITY_CPUS
+        -M pc,i8042=off
         -display $SERENITY_QEMU_DISPLAY_BACKEND
         -device $SERENITY_QEMU_DISPLAY_DEVICE
         -device virtio-serial,max_ports=2
         -device virtconsole,chardev=stdout
         -device isa-debugcon,chardev=stdout
         -device virtio-rng-pci
+        -machine vmport=off
```

Comment:
A USB/PS2 switch should be configurable via a `SERENITY_HID_HARDWARE={usb,ps2}` env var like we have with much other hardware.